src/config/dynamic-theme-fixes.config

```diff
@@ -12859,18 +12852,34 @@ a[class^="PaginationButton"]:hover {
 .nyh_navigation .nyh_submenu::before {
     border-bottom-color: var(--darkreader-bg--nyh-color-white) !important;
 }
+.nyh_teaser.nyh_teaser--story-grid,
+.nyh_icons_caret--factbox {
+    border: none;
+}
+.nyh_submenu-menucard,
 [class*="_Post__contentVisitor___"]::after {
     border-right-color: var(--darkreader-bg--nyh-color-grey-lighter) !important;
 }
 [class*="CurrentTopics__item___"],
 [class*="GroupSecondaryTeasers__showMoreButton___"],
 .nyh_mobile-menu__list-item,
+.nyh_menu-card__submenu-item,
 .nyh_submenu-menucard,
 .nyh_submenu {
     border-color: var(--darkreader-border--nyh-color-grey-lighter) !important;
 }
-.nyh_is-open .nyh_menu-card__link--mobile {
-    border: none !important;
+.nyh_fact-box {
+    border-bottom: 1px solid var(--darkreader-border--nyh-color-grey-lighter);
+    border-top: 1px solid var(--darkreader-border--nyh-color-grey-lighter);
+}
+.nyh_fact-box__body--closed::after {
+    background: linear-gradient(to bottom, #1e202180 0%, #1e2021 100%);
```

Comment:
`#1e202180`. I'm pretty sure this is not a correct hex color.

Reply:
`80` is for the opacity, so I get 50% transparency.